recipe/meta.yaml

```diff
@@ -8,7 +8,7 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://github.com/facebook/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: acf714d98e3db7b876e5b540cbf6dee298f60eb3c0723104f6d3f065cd60d6a8
+  sha256: 0d9ade222c64e912d6957b11c923e214e2e010a18f39bec102f572e693ba2867
 
 build:
   number: 1
```

Comment:
As we bumped version, build-number should be reset to zero

Reply:
reset